contracts/VaultV3.vy

```diff
@@ -727,11 +731,24 @@ def process_report(strategy: address) -> (uint256, uint256):
     # TODO: permissioned: ACCOUNTING_MANAGER (open?)
     return self._process_report(strategy)
 
+@external
+def sweep(token: address) -> (uint256):
+    # TODO: permissioned: ACCOUNTING_MANAGER
```

Comment:
let's add the ACCOUNTING_MANAGER

important to take into account when designing a contract that fills this role!

Reply:
i wasn't sure which role it should have been so i just marked it as accounting manager. i'll update and rebase now.